tests/tt_eager/python_api_testing/unit_testing/misc/test_optimized_conv_multi_core.py

```diff
@@ -152,24 +152,24 @@ def test_run_optimized_conv(
         # Run TT metal OP
         if not has_bias:
             bias_device = None
-        out = ttl.tensor.optimized_conv(
+        out = ttnn.operations.conv2d.optimized_conv(
```

Comment:
@arakhmati , should this become `ttnn.optimized_conv`?

Reply:
Ideally, the only API we should expose to the user is `ttnn.conv2d`. The reason I have not moved to the the ttnn namespace is because the user should not use `optimized_conv` directly.

Reply:
Yeah, only `ttnn.conv2d` should be exposed.